tensorflow/core/kernels/mkl_tfconv_op.h

```diff
@@ -101,13 +110,34 @@ class MklToTfOp : public OpKernel {
                                   input_number, output_shape, &output_tensor));
       CHECK_NOTNULL(output_tensor);
 
-      // Do we need to reorder Mkl layout into TensorFlow layout?
-      if (input.IsReorderNeeded(output_tf_pd)) {
-        // Insert reorder between Mkl layout and TensorFlow layout.
-        CHECK_EQ(input.CheckReorderToOpMem(output_tf_pd, output_tensor), true);
+      // Do we need to reorder MKL layout into TensorFlow layout?
+      if (input.IsReorderNeeded(OUTPUT_TF_MD)) {
+#ifdef ENABLE_MKLDNN_V1
+        std::vector<primitive> net;
+        std::vector<MemoryArgsMap> net_args;
+        // Insert reorder between MKL layout and TensorFlow layout.
+        // TODO(bhavanis): Refactor CheckReorderToOpMem() to directly insert
+        // the reorder primitive
+        bool status = input.CheckReorderToOpMem(output_tf_md, output_tensor,
```

Comment:
The statement can be inside `OP_REQUIRES`.  Same for the one below.

Reply:
Done